docs/docs/types/subscription.md

```diff
@@ -0,0 +1,39 @@
+A lite version of the form type this module focuses on gathering the minimum amount of user data necessary to maintain communication. By default only email address will be collected.
```

Comment:
I have nothing to say about this file that I haven't said elsewhere. Kudos for the consistency across docs though. It makes skimming way easier.